bin/main.ts

```diff
@@ -48,6 +48,7 @@ ssp.EksBlueprint.builder()
       machineImage: ec2.MachineImage.genericLinux({
         'us-east-1': 'ami-0b297a512e2852b89',
         'us-west-2': 'ami-06a8c459c01f55c7b',
+        'us-east-2': 'ami-07639d39e45f1a48d',
```

Comment:
Who maintains this AMI?

Reply:
this was leftover from testing. Will remove.

Reply:
Addressed.